### ISSUE TITLE
Configuration file with defaults

### ISSUE THREAD

**Issue by mgomersbach** [Lab-Brat/gentoo_update#28]
It could be that we add quite a few options and I guess some of us would like to have some defaults (in /etc/gentoo-update.conf or something)

Is this something you agree on and I can start working on?

**Comment by Lab-Brat** [comment]
Generally I think having a configuration file is a good idea.
But I just want to make sure I understand everything correctly first 😄

Right now, when we run `gentoo-update` the default behavior is to install security patches that `glsa-check` detects.
Will config file rewrite the defaults?

For example, if config file will be something like:
```ini
update_mode=full
read_logs=true
```
then running `gentoo-update` will be an equivalent of running `gentoo-update --update-mode full --read-logs`, right?

**Comment by mgomersbach** [comment]
Yes exactly, anything that can be set via flags could be set via the file
and vice versa.

**Comment by Lab-Brat** [comment]
> Is this something you agree on and I can start working on?

Sounds perfect! 
Feel free to implement it in the way you see fit 😸
